scripts/runtest.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
```

Comment:
What's that?

Reply:
It's a helper script for running unit tests in the debugger directly from VSCode. Breakpoints!